### PR TITLE
Improve READMEs

### PR DESCRIPTION
**Pull request by a-know**
refs https://github.com/mackerelio/go-check-plugins/pull/272

### CODE REVIEW

File: check-aws-cloudwatch-logs/README.md [hunk 1]
```diff
@@ -1,30 +1,68 @@
...
+Next, you can execute this program :-)
+
+```
+check-aws-cloudwatch-logs --log-group-name /aws/lambda/sample_log_group --pattern "Error" --critical-over 10 --warning-over 5
+```
+
+
+## Setting for mackerel-agent
+
+If there are no problems in the execution result, add s setting in mackerel-agent.conf .
```
Comment (itchyny) [inline check-aws-cloudwatch-logs/README.md]: What's `s` in `add s setting`?

Comment (a-know) [reply check-aws-cloudwatch-logs/README.md]: It's typo, fix 247c41a

**Comment by itchyny** [comment]
LGTM!

**Comment by a-know** [comment]
Thank you!

**Pull request merged**